components/Form/ObjectInput/ObjectInput.jsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import DynamicInput from 'components/FormWizard/DynamicInput';
+
+import styles from './ObjectInput.module.scss';
+
+const ObjectInput = ({
+  label,
+  hint,
+  name,
+  labelSize = 'm',
+  isInline,
```

Comment:
Could the default value be set to false?

Reply:
`undefined` is falsy, do we really need to specify a `false`? We are not doing it in any other place in the codebase

Reply:
It would be nice to not have to disable linting for this.
https://github.com/LBHackney-IT/lbh-social-care/blob/5ac4ddd3bbadac1f122d555f3fd49b3ba94d6f63/components/Form/Radios/Radios.jsx#L18

Reply:
Even if you have it the `eslint-disable-next-line no-unused-vars` is still needed because `  summaryInline` is used in another component and not in this one.

Reply:
I double-checked and the `isRadiosInLine` is the only `bool` prop with default value, I am still thinking that is going to be superfluous specify it to `false`.